src/wallet/wallet.ts

```diff
@@ -165,6 +178,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     this.xpriv = xpriv;
     this.seed = seed;
     this.xpub = xpub;
+    this.authPrivKey = authxpriv ? bitcore.HDPrivateKey(authxpriv) : null;
```

Comment:
Should we validate that `authxpriv` is a valid xprivkey and throw a proper error?

We can use [`bitcore.HDPrivateKey.isValidSerialized`](https://github.com/bitpay/bitcore/blob/88318365e65509a386376f39cd6b4579063cf654/packages/bitcore-lib/lib/hdprivatekey.js#L300)

Reply:
Done